content/posts/blog/2020-10-30-consuming-public-content.md

```diff
@@ -0,0 +1,108 @@
+
+## Longer-Term Gated Workflows
+
+The Docker TOS updates give us an opportunity to focus on the larger challenges with consuming public content. If you depend on public content, we recommend configuring a workflow that imports the content, security scans the content based on your organization's scanning policies, runs functional and integration tests to assure this most recent version of the content meets all expectations, then promote the validated content to a location your team(s) can utilize. The list of validations may start small, and evolve as new potential issues surface.
+
+We also recommend implementing a scheduled job that periodically checks for updates, including security updates to existing tags. And we further recommend to never let friends build against a :latest tag if at all possible
+
+![image alt text](/img/blog/2020-10-30-consuming-public-content/consuming2.png)
+
+Over the next few months, we will bring focus to gated workflows, and proposals for how we can enable increased consumption of public content with reduced risk of the public content causing outages or negative impacts. We believe a set of standards for properly processing public content through gated workflows will help elevate the consumption of the content while enabling healthy competition on how to best implement and support these gated workflows. Building gated workflows enables developers and project teams to take more risk with respect to the velocity of change for public content.
+
+Every public emergency (hurricanes, Covid-19, etc) reminds us of the importance of keeping critical supplies close to where we function. The content we store in Registries are just another example of the same local-cache model employed by Emergency Services Teams.
+
+## Cloud & ISV Specific Guidance
+
+Whether it’s a true exploit, well-intended security update, or the result of humans being human ... it’s not a matter of **if**, rather **when consuming public content will have some negative impact on your workflow.**
+
+To understand how to implement docker authenticated pulls, mitigating the Docker TOS changes, and how you can gate and buffer public content in your cloud provider, the following links are provided:
+
+* Docker™
+* [Azure™](https://aka.ms/consuming-public-content)
+* AWS™
```

Comment:
```suggestion
* AWS™
```

Reply:
As we're not putting out an AWS-specific blog, makes sense not to have this here.